src/main/resources/data/retail/pockets.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "name": "Travel Europe",
+    "icon": "travel",
```

Comment:
Did you check if these icon names will be the ones configured by default in the frontend?

Reply:
I am trying to get the default list...

Reply:
I used this list from Joao: default list of names/images [https://share.goabstract.com/cd3c6f04-5c35-403a-8941-018f71f14f9c]